tests/common.h

```diff
@@ -60,10 +60,25 @@ static inline void test_time_start(struct timespec* start) { clock_gettime(CLOCK
 void test_time_end(struct timespec* start, struct timespec* end, double* sum);
 
 /**
- * Generate a random trytes combination with given length
+ * @brief Initialize random number generator
+ *
+ * We use ASLR which stands for Address Space Layout Randomization, and we can assume each address of functions inside
```

Comment:
You don't have to mention ASLR in public header. But, you have to address its limitation.